File: 102B.cpp
#include<bits/stdc++.h>

using namespace std;

int ans = 0;
int digSum(int n) 
{ 
    ans += 1;
    if (n == 0)  
       return 0; 
    return (n % 9 == 0) ? 9 : (n % 9); 
} 

int main(){
    int input;
    cin >> input;
    digSum(input);
    cout << ans << endl;
}

File: 102B.py
a = input()
ans = 1
while(True):
    if(int(a)<10):
        ans = 0
        break
    sum = 0
    for i in str(a):
        sum += int(i)
    a = sum
    if(a<10):
        break
    else:
        ans += 1
print(ans)